templates/pkg/resource/descriptor.go.tpl

```diff
@@ -32,7 +33,7 @@ type resourceDescriptor struct {
 // GroupKind returns a Kubernetes metav1.GroupKind struct that describes the
 // API Group and Kind of CRs described by the descriptor
 func (d *resourceDescriptor) GroupKind() *metav1.GroupKind {
```

Comment:
Do we still need this method?

Reply:
Yes, because the runtime doesn't import the resource package itself (it loads the resource manager factory from a registry) but still needs to know the GroupKind:

https://github.com/aws-controllers-k8s/runtime/blob/8e4dc5ecc41f9370e413c3be4ee88baf67ecbcd1/pkg/runtime/reconciler.go#L65-L72